lib/tasks/import_data.rake

```diff
@@ -23,7 +23,7 @@ task :import_data => :environment do
   end
 
   # TODO: verify that there are the proper number of resources once we have all the state data
-  verify(Resource, 1)
+  verify(Resource, 9)
```

Comment:
I'm going to update this to actually count how many resources should be present rather than just having to update the number as I create more seed data lol